rpms/pgp-zimlet.spec

```diff
@@ -36,16 +36,21 @@ cp -R tk_barrydegraaff_zimbra_openpgp/tk_barrydegraaff_zimbra_openpgp.zip $RPM_B
 
 %post
 su - zimbra -c "zmzimletctl deploy /opt/zimbra/zimlets-extra/tk_barrydegraaff_zimbra_openpgp.zip"
+su - zimbra -c "zmprov fc all"
+
 
 %preun
 su - zimbra -c "zmzimletctl undeploy tk_barrydegraaff_zimbra_openpgp"
+su - zimbra -c "zmprov fc all"
```

Comment:
It seems the caching issue has not been solved with this command.

@barrydegraaff please do tests and tell me if it works or not at your side.